java/dbschema/src/org/netbeans/modules/dbschema/jdbcimpl/IndexElementImpl.java

```diff
@@ -95,7 +95,7 @@ public ColumnElement getColumn(DBIdentifier name) {
 		return (ColumnElement) columns.find(name);
     }
   
-    protected void initColumns(LinkedList idxs) {
+    protected void initColumns(List idxs) {
```

Comment:
This is an incompatible change. I don't see enough value to risk breakage (this is friendonly, but changing this is not friendly).

Reply:
done..